ze/btx_zeinterval_callbacks.hpp

```diff
@@ -51,6 +69,12 @@ using btx_event_desct_t =
 using btx_command_list_desc_t =
     std::tuple<ze_command_queue_desc_t, ze_device_handle_t, bool /*hCommandListIsImmediate*/>;
 
+struct DeviceHash {
+    uint64_t hash;
+    uint32_t deviceIdx;
+};
```

Comment:
If you never mutate member of `DeviceHash`I would prefer to use tuple just because we use them everywhere else

Reply:
Used tuple instead.